src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -720,6 +721,14 @@ registerVisualizationLayerDefinition({
     if (!model.showTransformSelection) {
       return;
     }
+
+    if (
+      !!positionedGlyph.varGlyph?.glyph.customData["fontra.glyph.locked"] ||
```

Comment:
I played with this, but this is not the right thing: it only hides the visualization, but you still see the cursor change, and you can still try to drag the (now invisible) handles.

Reply:
Thanks for the feedback and testing. I did not notice that behavior. I removed this part of the code.